doc/dev/autograph.rst

```diff
@@ -898,6 +898,38 @@ the context will be converted anyways:
 ...     return x
 
 
+Adding modules for Autograph conversion
+---------------------------------------
+
+Library code is not meant to be targeted by Autograph conversion, hence 
+``pennylane``, ``catalyst`` and ``jax`` modules have been excluded from it.
+But sometimes it might make sense enabling specific submodules from the 
+excluded modules for which conversion may be appropriate. For this cases 
+one can use the ``autograph_include`` parameter, which provides a list 
+of modules/submodules that will always be enabled for conversion no matter
+if the default conversion rules were excluding them before.
+
+This is how you can enable previously excluded modules:
+
+>>> import excluded_module_1
+... import excluded_module_2
+...
+... @qjit(autograph=True, autograph_include=["excluded_module_1", "excluded_module_2"])
+... def g(x: int):
+...     return excluded_module_1.f(x) + excluded_module_2.f(x)
+
+And this example shows how you can enable a previously excluded submodule:
+
+>>> import excluded_module
+...
+... @qjit(autograph=True, autograph_include=["excluded_module.submodule"])
+... def g(x: int):
+...     return excluded_module.submodule.f(x)
```

Comment:
Since these two examples are essentially the same, and we don't want to encourage the top one, I would suggest only using the bottom one.

Reply:
Makes sense, yes. Done.